core/commands/files.go

```diff
@@ -829,6 +843,13 @@ Example:
 
     $ ipfs files mv /myfs/a/b/c /myfs/foo/newc
 
+WARNING:
+
+Use caution when calling commands with '--flush=false'.
+Passing '--flush=false' may improve performance for large numbers of file
+operations, but it does so at the cost of consistency guarantees and unbound
+growth of the directories' in-memory caches.
+Make sure to call 'ipfs files flush' after each batch of work.
```

Comment:
If this warning text is identical for the different commands, please define a `const` and use that in the help text. Ex: https://go.dev/play/p/8CP7Sz13CzD